demo/JD-recommendation/sync_select_out.sql

```diff
@@ -0,0 +1,88 @@
+USE JD_db;
+set @@sync_job='true';
+set @@job_timeout='600000';
```

Comment:
here timeout=600000ms, may meet with timeout error when exporting the whole JD dataset, but should be fine with small data